modules/ROOT/pages/cross_cutting/oauth2_proxy.adoc

```diff
@@ -0,0 +1,70 @@
+:imagesdir: ../images
+= OAuth2 Proxy
```

Comment:
The level one heading should be more descriptive like "Authentication with Keycloak using a OAuth2 Proxy"